templates/projects/update.html

```diff
@@ -62,6 +62,14 @@ <h3>{{_('Enable project syncing')}}</h3>
         </div>
     </fieldset>
 </form>
+<br>
+<form id="remove-password-form" method="post" action="{{ url_for('project.remove_password', short_name = project.short_name) }}">
```

Comment:
This can probably include a conditional per [here](https://github.com/bloomberg/pybossa/pull/945/files/6f65387fad29337aa7c35a5e7cb2d0b23457e1d0#diff-7fa4bf701959a15b1589099598a22b2352b452e47a90c7b981d5d340d8bc45dfR957):

```html
{% if config.PROJECT_PASSWORD_REQUIRED %}
  <form id="remove-password-form" method="post" action="{{ url_for('project.remove_password', short_name = project.short_name) }}">
      <div class="form-actions">
          <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
          <input type="hidden" name="password" value="" />
          <button type="submit" class="btn btn-link" value="Remove Project Password">{{_('Remove Project Password')}}</button>
      </div>
  </form>
{% endif %}
```

Reply:
Great catch! Implemented via b613336